sass/includes/_event-tags.scss

```diff
@@ -0,0 +1,24 @@
+.event-tags {
+    display: inline-flex;
+    align-self: flex-start;
+    list-style-type: none;
+    flex-basis: 50%;
+    border: 2px solid colour.brand-colour("pastel-pink");
```

Comment:
There is a helper for this:

```scss
@include colour.colour-border("pastel-pink", 2px);
```

Reply:
I have added `@use "@nationalarchives/frontend/nationalarchives/tools/colour";`

Then the border property as you suggest above. 

This is what I see in the page

<img width="323" alt="Screenshot 2023-10-19 at 13 03 38" src="https://github.com/nationalarchives/ds-wagtail/assets/771869/a42f2298-76d1-45af-9cfc-5f88bd0b127b">


<img width="325" alt="Screenshot 2023-10-19 at 13 03 06" src="https://github.com/nationalarchives/ds-wagtail/assets/771869/f759e996-2ec8-4c49-bcb8-9532d881ea39">

`var(--pastel-pink)` appears not to be defined.

Is there another sass file I need to `@use`?

Reply:
Nope. Revert that - that's just highlighted an issue with using brand colours in the colour mixins...

As you were! 😅